src/DrCabal/Profile.hs

```diff
@@ -149,6 +159,7 @@ formatChart start end width libs = unlines $ concat $
         , "  " <> fmt [cyan]   block <> "  Downloading"
         , "  " <> fmt [blue]   block <> "  Starting"
         , "  " <> fmt [red]    block <> "  Building"
+        , "  " <> fmt [green]  block <> "  Haddock"
```

Comment:
Should it hide the haddock if it was not used?

Reply:
I think it's fine to always keep it 👍🏻 
Doesn't take too much space and gives a good indication of what else can be measured 🙂